typescript/nomad-sdk/src/nomad/domains/staging.ts

```diff
@@ -1,43 +1,43 @@
 import { NomadDomain } from './domain';
 
 export const kovan: NomadDomain = {
-  "name": "kovan",
-  "id": 3000,
-  "paginate": {
-    "from": 29193952,
-    "blocks": 2000,
+  name: 'kovan',
+  id: 3000,
+  paginate: {
+    from: 29329838,
+    blocks: 2000,
   },
-  "home": "0xA2d62FbcD779231894586CFC569E8a085662d5Bb",
-  "replicas": [
+  home: '0x134610222e796deAdDe52Fd2E4710D55A32899d5',
+  replicas: [
     {
-      "domain": 5000,
-      "address": "0x2fFEa0eBe27cF63cbE4b8bb621b817F7fe40f27B",
+      domain: 1634956402,
+      address: '0x044d4615e14237888669C918b3d0069bD6682905',
     },
   ],
-  "governanceRouter": "0x2960C101Cb72eCC35a515e247608b17B46f38bA6",
-  "bridgeRouter": "0x983fC3130002454d0e892ae7a53147B09Fb58fd8",
-  "tokenRegistry": "0x60fB0Ea196F2c0Ba54FBFAB4184cCFb2ce4048BD",
-  "ethHelper": "0xe3717ed5f3F087f3AAbaD4C25ee924EDeB80c9f6",
+  governanceRouter: '0x3B58BE35841176f00684af0e1Cc59D88c8e0847c',
+  bridgeRouter: '0x9aE2c4CA2ABaeE4f3C74057326686D27F0B44d68',
+  tokenRegistry: '0x7D27AC216F7239260fd3594f265401977fea69f9',
+  ethHelper: '0x77d3406cE820A4740906ca4849ad7e38B896EfF8',
 };
 
-export const moonbasealpha: NomadDomain = {
-  "name": "moonbasealpha",
-  "id": 5000,
-  "paginate": {
-    "from": 1507707,
-    "blocks": 2000,
+export const astar: NomadDomain = {
```

Comment:
undo when we shut down staging demo

Reply:
Yeah everything done in this PR will be deleted. Only here temporarily